setup/baseConfig.yaml

```diff
@@ -102,6 +102,7 @@ file_cache:
   max-size-mb: <maximum cache size allowed. Default - 80% of free disk space>
   high-threshold: <% disk space consumed which triggers eviction. This parameter overrides 'timeout-sec' parameter and cached files will be removed even if they have not expired. Default - 80>
   low-threshold: <% disk space consumed which triggers eviction to stop when previously triggered by the high-threshold. Default - 60>
+  usage-from-cache: true|false <use file cache to report drive size and storage statistics (StatFs)>
```

Comment:
Should we clarify that this should only be false when using Seagate Lyve Cloud?

Reply:
Yeah, I can add that note.
We can also discuss the various options for this flag (default the other way? Or move the flag to the s3storage component?)
I'm starting to think it might need to move to s3storage...